json-schema-core/src/main/java/io/ballerina/jsonschema/core/SchemaUtils.java

```diff
@@ -0,0 +1,42 @@
+package io.ballerina.jsonschema.core;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.ToNumberPolicy;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SchemaUtils {
+    public static final List<String> SUPPORTED_DRAFTS = List.of("https://json-schema.org/draft/2020-12/schema");
+
+    static final Map<String, String> ID_TO_TYPE_MAP = new HashMap<>();
```

Comment:
Shall we add a small comment saying this needs for reference to get identify the type